cypress/support/fragments/inventory/inventoryMainButton.js

```diff
@@ -17,4 +19,29 @@ export default {
   waitModalLoading(modalName) {
     cy.expect(Modal(modalName).exists());
   },
+  createNewRecordByShortcuts() {
+    cy.get('body').type('{alt}N');
+  },
+  editRecordByShortcuts() {
+    cy.get('body').type('{ctrl}{alt}E');
+  },
+  saveRecordByShortcuts() {
+    cy.get('body').type('{ctrl}S');
+  },
+  openShortcutsByHotkey() {
+    cy.get('body').type('{ctrl}{alt}K');
+  },
+  closeShortcutsByHotkey() {
+    cy.get('body').type('{esc}');
+  },
```

Comment:
I suggest to use general object for hot keys like this

```
const hotKeys = {
    create: '{alt}N',
    edit: '{ctrl}{alt}E',
    save: {ctrl}S
    ...
}
```

And create only one method for typing with parameters
Instead of editRecordByShortcuts, openShortcutsByHotkey and others you will have only one:

```
pressHotKey(hotKey) {
    cy.get('body').type(hotKey);
  },
```

You can use it with this way
```
pressHotKey(hotKeys.new)
pressHotKey(hotKeys.edit)
```